### ISSUE TITLE
Add Client-Matter to Document Names

### ISSUE THREAD

**Issue by JulieJager** [syncids/syncids-ui#208]
Currently, in the new UI, our IDS and references folder do not include the client-matter in the name.  

![capture](https://cloud.githubusercontent.com/assets/17907226/15400314/ad866d4e-1db0-11e6-8776-3c0b32359267.JPG)

When a user clicks "Extract All" the prompt points to a preexisting folder (e.g., Documents,Downloads, etc.) and proposes a name of the ZIP folder its going to save (we don't want another zip folder).  

![capture](https://cloud.githubusercontent.com/assets/17907226/15400400/00724d70-1db1-11e6-82bf-e29acc6d0449.JPG)

We originally thought it was creating a FOLDER with the client-matter name, but it is not - its just another ZIP Folder.  Users cannot open the ZIP when trying to upload each document inside to EFS-WEB.  EFS-Web grabs the entire ZIP Folder.  

Browsing to another folder after clicking "Extract all" doesn't work either because then user loses the client-matter information.  This is hard to explain in writing.  I will demo the problem on our next call.

Bottom line:  I think we need to tack on the Client-Matter info as shown below:

![idsformss](https://cloud.githubusercontent.com/assets/17907226/15399652/291fb0b2-1dae-11e6-84cd-85763a51786a.JPG)

This way, it doesn't matter which folder our users save our documents to, the client-matter name will always be there.  If we do this, we will not need the .txt document.
